web/index.html

```diff
@@ -47,6 +47,26 @@
         ::-ms-reveal {
             display: none;
         }
+        @media screen and (min-width: 768px) {
+            .splash-logo {
+                width: 104px;
+                height: 104px;
+            } 
+        }
+
+        @media screen and (max-width: 768px) {
```

Comment:
Pretty confused by this ... if we have `min-width: 768px` and `max-width: 768px`, and those are applying the same style, why not just do something like this?

```css
@media screen and (max-width: 479px) {
    .splash-logo {
        width: 52px;
        height: 52px;
    }
}

/* This covers all screens wider than 479px */
@media screen and (min-width: 480px) {
    .splash-logo {
        width: 104px;
        height: 104px;
    }
}
```

Reply:
Fair point